src/index.js

```diff
@@ -71,6 +72,10 @@ export default function (debug = printDebug) {
       })
     }
 
+    function buildAssertionsOutOfWrapperObject(wrapperObject) {
+      Object.keys(wrapperObject).forEach(key => overwriteMethod(wrapperObject[key], key))
```

Comment:
Object.keys and forEach are actually ES5 and supported everywhere http://kangax.github.io/compat-table/es5/#Object.keys. So I fail to see the benefit but if you want to see https://www.npmjs.com/package/foreach and https://www.npmjs.com/package/object-keys used instead, sure I'll change it to get this merged.